chart/templates/scheduler/scheduler-deployment.yaml

```diff
@@ -53,7 +53,6 @@ metadata:
     release: {{ .Release.Name }}
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     heritage: {{ .Release.Service }}
-    executor: {{ .Values.executor }}
```

Comment:
I think we can sanitize the value or add it when it contains only acceptable characters and when it is shorter than 64 characters.

Reply:
In my humble opinion, sanitizing this value is a little too much work (complexity) for this field that has no functional use. I'd prefer to remove it then making it too complex...